src/components/Suggested.jsx

```diff
@@ -0,0 +1,53 @@
+import React, { useState, useEffect } from 'react';
+import ReactDOM from 'react-dom';
+
+function Suggested() {
+
+  let counter = 1;
+  const [suggestionItems, updateSuggestionItems] = useState([]);
+
+  function get() {
+    $.ajax({
+      context: this,
+      method: 'GET',
+      url: '/get/random',
+      success: function(result) {
+        if (counter === 2) {
+          console.log(result);
+          updateSuggestionItems(result);
+        }
+      },
+      error: function(err) {
+        console.error('There was an error getting suggested items from database: ', err);
+      }
+    });
+  }
+
+  useEffect(() => {
+    if (counter === 1) {
+      counter += 1;
+      get();
+    }
+  }, [counter]);

```

Comment:
is the purpose of this code to only work w/ data from the 2nd get request? if so, what purpose does that serve?